pkg/glassnode/glassnodeapi/derivatives.go

```diff
@@ -1,4 +1,4 @@
-package glassnode
+package glassnodeapi
```

Comment:
maybe move them into pkg/datasource/glassnode ? so that we can integrate other data sources?